spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/TracingProperties.java

```diff
@@ -53,4 +74,81 @@ public void setProbability(float probability) {
 
 	}
 
+	public static class Baggage {
```

Comment:
This class could have an `enabled` property, removing the need for some additional metadata.